### PR TITLE
FI-2833 Add R5 Support

### PR DESCRIPTION
**Pull request by 360dgries**
Added support for FHIR R5:
- Updated generator
- Setup `excluded_files`
- Setup R5 module and version-specific files (model, json, xml, etc.)

### CODE REVIEW

**Review by Jammjammjamm — COMMENTED** [review]
This PR seems to contain all of the changes from the previous PR as well.

File: lib/fhir_models/r5/generated/resources/Account.rb [hunk 1]
```diff
@@ -0,0 +1,210 @@
+module FHIR
+  module R5
+    class Account < Model
+      include FHIR::Hashable
+      include FHIR::Json
+      include FHIR::Xml
+
+      SEARCH_PARAMS = []
```
Comment (Jammjammjamm) [inline lib/fhir_models/r5/generated/resources/Account.rb:8]: This is populated for other versions.

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -30,12 +30,9 @@ Layout/EmptyLinesAroundModuleBody:
 Layout/TrailingEmptyLines:
   Exclude:
     - 'lib/fhir_models/bootstrap/json.rb'
-    - 'lib/fhir_models/r4/fhir_ext/structure_definition.rb'
-    - 'lib/fhir_models/r4/fhir_ext/structure_definition_compare.rb'
-    - 'lib/fhir_models/r4/fhir_ext/structure_definition_finding.rb'
-    - 'lib/fhir_models/r4b/fhir_ext/structure_definition.rb'
-    - 'lib/fhir_models/r4b/fhir_ext/structure_definition_compare.rb'
-    - 'lib/fhir_models/r4b/fhir_ext/structure_definition_finding.rb'
+    - 'lib/fhir_models/*/fhir_ext/structure_definition.rb'
```
Comment (Jammjammjamm) [inline .rubocop_todo.yml:33]: Does this file even exist anywhere?

Comment (yunwwang) [reply .rubocop_todo.yml]: fil eupdated by rerun `rubocop --auto-gen-config`

File: lib/fhir_models/r5/generated/resources/Account.rb (hunk 1 above)
Comment (yunwwang) [reply lib/fhir_models/r5/generated/resources/Account.rb]: Fixed. The problem is that SearchParameter.xpath is deprecated in R5. This also fixed some missing search parameters in R4!

File: .rubocop_todo.yml [hunk 3]
```diff
@@ -32,19 +15,10 @@ Layout/TrailingEmptyLines:
...
-
-# Offense count: 3
-# Cop supports --auto-correct.
-# Configuration parameters: AllowInHeredoc.
-Layout/TrailingWhitespace:
-  Exclude:
-    - 'lib/fhir_models/fhir_ext/common_structure_definition.rb'
-    - 'lib/fhir_models/fhir_ext/common_structure_definition_compare.rb'
-    - 'lib/fhir_models/fhir_ext/common_structure_definition_finding.rb'
+    - 'lib/fhir_models/r5/fhir_ext/structure_definition.rb'
```
Comment (Jammjammjamm) [inline .rubocop_todo.yml:20]: Some of these files still don't exist.

Comment (yunwwang) [reply .rubocop_todo.yml]: This is auto generated. Maybe we should remove this file? It does not anything. I manually removed some lines as those are not there.

**Review by Jammjammjamm — APPROVED** [review]

**Pull request merged**